docs/upgrade/v1-2-2-to-v1-3-1.md

```diff
@@ -21,6 +21,13 @@ For air-gapped environments, see [Prepare an air-gapped upgrade](./automatic.md#
 
 ### 1. Cluster upgrade stuck after the first node is upgraded
 
+:::note
+For users running v1.2.2/v1.3.0, labelling the `local-kubeconfig` secret before triggering the upgrade, could potentially avoid the subsequent stuck upgrade issue. 
+
+This can be done as follows: 
+`kubectl label secret local-kubeconfig -n fleet-local cluster.x-k8s.io/cluster-name=local`
+:::
```

Comment:
```suggestion
:::tip
To prevent this issue from occurring, label the `local-kubeconfig` secret before starting the upgrade process. 
`kubectl label secret local-kubeconfig -n fleet-local cluster.x-k8s.io/cluster-name=local`
:::
```

Reply:
This note should be placed after the example so that the context is set first.